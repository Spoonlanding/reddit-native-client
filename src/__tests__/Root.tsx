import 'react-native';
import * as React from 'react';
import { shallow } from 'enzyme';

// tslint:disable-next-line:import-name
import Root from '../index';

describe('<Root />', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<Root />);
  });

  it('renders AppLoading before fonts have loaded', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders component when fonts have loaded', () => {
    wrapper.setState({ isReady: true });
    expect(wrapper).toMatchSnapshot();
  });
});
