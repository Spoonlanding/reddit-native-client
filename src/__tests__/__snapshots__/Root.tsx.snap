// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> renders as expected 1`] = `<AppLoading />`;
